pvlib/pvsystem.py

```diff
@@ -164,15 +163,14 @@ class PVSystem:
 
     inverter : string, optional
         The model name of the inverters.
-        May be used to look up the inverter_parameters dictionary
-        via some other method.
 
     inverter_parameters : dict or Series, optional
         Inverter parameters as defined by the SAPM, CEC, or other.
 
-    racking_model : string, default 'open_rack'
-        Valid strings are 'open_rack', 'close_mount', and 'insulated_back'.
-        Used to identify a parameter set for the SAPM cell temperature model.
+    racking_model : string, default None
```

Comment:
Unless `None` is a meaningful default value (rather than merely a placeholder indicating that a meaningful value was not provided), we should say `optional` instead.  #1574